crates/weaver_semconv/src/v2/entity.rs

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! The new way we want to define entities going forward.
+
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    group::GroupSpec,
+    v2::{attribute::AttributeRef, CommonFields},
+};
+
+/// An EntityGroup defines a new entity.
+#[derive(Serialize, Deserialize, Debug, Clone, JsonSchema)]
+#[serde(deny_unknown_fields)]
+pub struct EntityGroup {
+    /// The type of the Entity.
+    pub r#type: String,
+    /// The attributes that make the identity of the Entity.
+    pub identity: Vec<AttributeRef>,
+    /// The attributes that make the description of the Entity.
+    #[serde(default)]
+    #[serde(skip_serializing_if = "Vec::is_empty")]
+    pub description: Vec<AttributeRef>,
```

Comment:
Just a suggestion to clarify the semantic of these fields:
- `identity` -> `identifying_attributes` or `id_attributes`
- `description` -> `descriptive_attributes` or `info_attributes`